### PR TITLE
Rake task to modify birthdates by parameter

### PR DESCRIPTION
**Pull request by MarcosFierro**

### CODE REVIEW

File: lib/tasks/play.rake [hunk 1]
```diff
@@ -0,0 +1,5 @@
+desc "Print reminder about eating more fruit."
+
+task :play do
+  puts "Eat more apples!"
+end
```
Comment (guillezorron) [inline lib/tasks/play.rake]: Please delete it, i don't like apples :sweat_smile:

**Comment by guillezorron** [comment]
When we were thinking in this card we are talking about add birth date for each user and after that use it to calculate the age of each user, it is a bit more difficult.. but i don't know what @AlvaroSenorale think about that

File: lib/tasks/birthdates.rake [hunk 2]
```diff
@@ -0,0 +1,7 @@
+desc "Sets users birthdates"
+
+namespace :update_birthdates do
+  task :to, [:date] => [:environment] do |task, args|
+    User.update_all birth_date: args[:date]
```
Comment (AlvaroSenorale) [inline lib/tasks/birthdates.rake]: could we add a check to validate that the data passed by args is well formatted ? I was thinking on something like 
```ruby
Time.parse(args[:date])
# for example
Time.parse('12-334-34') # will raise "ArgumentError: argument out of range"
Time.parse('asdasdasd') # will raise "ArgumentError: no time information.."
```

**Comment by AlvaroSenorale** [comment]
@guillezorron is right, Maybe we could add a bit of realism on the birth date, instead of making all users to have same birth date. We would need to iterete over user collection and for each one pick a random date in a time range that makes sense. What do you think @MarcosFierro  ?

**Comment by MarcosFierro** [comment]
Make sense, i can do that!

File: lib/tasks/birthdates.rake (hunk 2 above)
Comment (MarcosFierro) [reply lib/tasks/birthdates.rake]: Good, i add that prevention

**Comment by MarcosFierro** [comment]
I created two more tasks, one that assigns random birth_dates to all users and the other one to calculate user's age up to the date

File: lib/tasks/birthdates.rake [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  def rand_in_range(from, to)
+    rand * (to - from) + from
+  end
+  
+  def rand_time(from, to=2.years.ago)
+    Time.at(rand_in_range(from.to_f, to.to_f))
+  end
+  
+  task :random => [:environment] do |task, args|
+    users_updated = User.all.each { |user| user.birth_date = rand_time((60*365).days.ago) }
```
Comment (AlvaroSenorale) [inline lib/tasks/birthdates.rake]:
```ruby
User.find_each do |user|
  birth_date = rand_time((60*365).days.ago)
  user.update(birth_date: birth_date)
end

Comment (MarcosFierro) [reply lib/tasks/birthdates.rake]: Done, thanks!

**Pull request merged**